### PR TITLE
PERF-5711 remove quiesce so we can rolling restart

### PR DESCRIPTION
**Pull request by hilldani**
**Jira Ticket:** PERF-5711

### Whats Changed
Replace quiesce with noop which appears to not impact the results in this test thankfully. This lets us then use this genny test to test rolling restarts. Instead of making genny quiesce handle a rolling restart we should keep this in mind for Locust

### Patch Testing Results
[patch](https://spruce.mongodb.com/version/66a9417b7f65a40007ac9e15/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC)

### CODE REVIEW

**Review by lgfang — COMMENTED** [review]
After the code change, the whole `QuiesceBetweenLevels` does nothing. Can we just remove this actor?

**Comment by hilldani** [comment]
> After the code change, the whole `QuiesceBetweenLevels` does nothing. Can we just remove this actor?

good point. removed and retested in the patch

File: src/workloads/scale/MixedWorkloadsGenny.yml [hunk 1]
```diff
@@ -128,61 +128,44 @@ Actors:
         Indexes:
           - keys: {id: 1}
           - keys: {a: 1}
-      - Phase: 1..10
+      - Phase: 1..6
```
Comment (jawwadasghar) [inline src/workloads/scale/MixedWorkloadsGenny.yml:131]: The max phase is 5 no?

File: src/workloads/scale/MixedWorkloadsGenny.yml [hunk 2]
```diff
@@ -102,7 +102,7 @@ ActorTemplates:
       Phases:
         OnlyActiveInPhases:
           Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: 1}}]
-          NopInPhasesUpTo: 10
+          NopInPhasesUpTo: 6
```
Comment (jawwadasghar) [inline src/workloads/scale/MixedWorkloadsGenny.yml:105]: If max phase is indeed 5, would change this to reflect that

**Comment by jawwadasghar** [comment]
Does it make sense to run a multipatch to see if removing the quiesce increases CV?

**Review by jawwadasghar — CHANGES_REQUESTED** [review]
Would suggest a multipatch in case removing quiesce adds noise. 
Might be fine even if it does, but would be good to know the extent if any. 
Also think the max phase is 5 rather than 6.

**Review by jawwadasghar — APPROVED** [review]
LGTM

**Pull request merged**